pom.xml

```diff
@@ -240,6 +241,11 @@
 				<artifactId>coreui__ajax</artifactId>
 				<version>1.0.6</version>
 			</dependency>
+			<dependency>
+				<groupId>com.vladsch.flexmark</groupId>
+				<artifactId>flexmark-all</artifactId>
+				<version>${flexmark.version}</version>
```

Comment:
No need to add all versions as properties. I recommend to have as properties versions only for sets of libs. In this case it's uniq lib usage.

Reply:
Understood. Thanks